tests/test_mediaservice.py

```diff
@@ -0,0 +1,19 @@
+# Tests for the openslides media service
+import requests
+import time
+
+def test_mediaservice_positiv():
+    # wait the service to start
+    time.sleep(10)
```

Comment:
Hi,
plase add an entrypoint to the Dockerfile.tests, that executes a `wait-for-it` for the media service and the database. So the tests only start, if everything is ready.